lib/WP_Auth0_LoginManager.php

```diff
@@ -34,7 +34,7 @@ class WP_Auth0_LoginManager {
 	 * @param WP_Auth0_UsersRepo    $users_repo - see member variable doc comment.
 	 * @param WP_Auth0_Options|null $a0_options - see member variable doc comment.
 	 */
-	public function __construct( WP_Auth0_UsersRepo $users_repo, WP_Auth0_Options $a0_options ) {
+	public function __construct( WP_Auth0_UsersRepo $users_repo, ?WP_Auth0_Options $a0_options ) {
```

Comment:
Oh noooooooo!
This is a syntax error in PHP 7.0

Reply:
Does null ever get passed as argument?

Reply:
It looks like that wasn't caught in v4.0.0. You used to be able to pass null but not any longer. Nothing in that class would work without that property.

Reply:
Done.